go/service/policies/api_op_CreatePolicy_test.go

```diff
@@ -99,10 +97,11 @@ func Test_CreatePolicyInternalServerError(t *testing.T) {
 	apiClient, err := New(*cfg)
 	assert.NoError(t, err, "failed to initialize api client")
 
-	createdPolicy, apiError, err := apiClient.CreatePolicy(CreatePolicyInput{})
+	createdPolicy, err := apiClient.CreatePolicy(CreatePolicyInput{})
 
-	assert.NoError(t, err, "failed to list policies.")
-	assert.NotNil(t, apiError)
 	assert.Nil(t, createdPolicy)
-	assert.Equal(t, len(apiError), 1)
+	assert.NotNil(t, err)
+
+	assert.Contains(t, err.Error(), "connector api error", "error message should contain 'connector api error'")
```

Comment:
Let's make the `connector api error` a const so that we can use it easily without hardcoding it.

Reply:
I fixed these so I'll mark as resolved as they're a minor fix